### PR TITLE
System export

### PR DESCRIPTION
**Pull request by sh3nan1gans**

### CODE REVIEW

File: datacalls.go [hunk 1]
```diff
@@ -10,6 +10,19 @@ const (
...
+type Collection struct {
+	SystemKey    string
+	CollectionID string
+	Name         string
+	Columns      []Column
+	Users        bool
+}
+
+type Column struct {
+	name       string
```
Comment (cwandrews) [inline datacalls.go]: Any reason why ColumnType is exported and name isn't? also naming feels inconsistent here @sh3nan1gans @aallsbrook

File: datacalls.go [hunk 2]
```diff
@@ -134,3 +147,19 @@ func deletedata(c cbClient, collection_id string, query *Query) error {
 	}
 	return nil
 }
+
+func (d *DevClient) GetColumns(collection_id string) ([]interface{}, error) {
```
Comment (cwandrews) [inline datacalls.go]: This is a thing that both Dev's and User's can do, so I think there should be an implementation for UserClient too. @sh3nan1gans @aallsbrook

File: devcalls.go [hunk 3]
```diff
@@ -261,6 +261,59 @@ func (d *DevClient) GetCollectionInfo(collection_id string) (map[string]interfac
...
+	//fmt.Printf("body: %+v\n", resp.Body)
+	return resp.Body.([]interface{}), nil
+}
+
+func (d *DevClient) GetAllRoles(SystemKey string) ([]interface{}, error) {
+	creds, err := d.credentials()
+	if err != nil {
+		return nil, err
+	}
+	resp, err := get("/admin/user/"+SystemKey+"/roles", map[string]string{
```
Comment (cwandrews) [inline devcalls.go]: for admin calls these should be using the constants to build the urls like _DEV_PREAMBLE @sh3nan1gans @aallsbrook

File: datacalls.go (hunk 1 above)
Comment (sh3nan1gans) [inline datacalls.go]: Looks like we aren't actually using these types. I'm going to remove them.

Comment (cwandrews) [inline datacalls.go]: On 2/25/15 3:20 PM, sh3nan1gans wrote:
> Looks like we aren't actually using these types. I'm going to remove them.
Ok sounds good.

**Pull request merged**